code/modules/mob/new_player/new_player.dm

```diff
@@ -73,7 +74,7 @@
 
 	output += "</div>"
 
-	var/datum/browser/popup = new(src, "playersetup", "<div align='center'>New Player Options</div>", 240, 300)
+	var/datum/browser/popup = new(src, "playersetup", "<div align='center'>Welcome to TGMC - [SSmapping.configs[SHIP_MAP].map_name]</div>", 560, 325)
```

Comment:
```suggestion
	var/datum/browser/popup = new(src, "playersetup", "<div align='center'>Welcome to TGMC[SSmapping?.configs ? "- [SSmapping.configs[SHIP_MAP].map_name]" : ""]</div>", 560, 325)
```
Test this. The current setup is runtiming because at the start `SSmapping` is not yet up.